src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -487,7 +487,7 @@ extern "C" {
 /**
  * Version of SignalingClientInfo structure
  */
-#define SIGNALING_CLIENT_INFO_CURRENT_VERSION 1
+#define SIGNALING_CLIENT_INFO_CURRENT_VERSION 2
```

Comment:
Since, we are updating the struct we would need to update the version table here: https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/develop/src/source/Signaling/Signaling.c#L586
Basically need to set the defaults for versions less than 2 and need internal copy. Signaling client info maintain an internal copy of all the variables, so need to do the same for this and so that the updated pSignalingClient version is passed around.